src/main/resources/assets/farmersdelight/lang/de_de.json

```diff
@@ -4,14 +4,14 @@
   "block.farmersdelight.basket": "Korb",
   "block.farmersdelight.cutting_board": "Schneidbrett",
   "block.farmersdelight.skillet": "Bratpfanne",
-  "block.farmersdelight.oak_pantry": "Eichenholzküchenschrank",

```

Comment:
It seems you used the new 1.17+ keys in this lang. 1.16 still uses the old keys, such as `pantry` and `rice_crop`.
All diff lines that change keys are affected, so it should be easy to rollback.

Reply:
they should now be back to the old ones didn't notice that they stayed untouched in 1.16